rubi/rubi/client.py

```diff
@@ -585,6 +591,27 @@ def batch_cancel_limit_orders(self, transaction: Transaction) -> TransactionRece
             ids=order_ids,
             **transaction.args(),
         )
+    
+    ######################################################################
+    # data methods (raw data) TODO: once we have cleanly formatted data functions, we can switch to only exposing those
+    ######################################################################
+
+    def get_offers(
```

Comment:
While I think this is fine as a first parse in the future I think this should conform to the pattern that has been established in the client by using `pair_name` instead of `pay_gem` and `buy_gem`.

also what is `from_address`?

Reply:
agreed, will be easier for users to have this type of semantic congruity across functions, will iterate this to utilize `pair_name`.

`from_address` is the address that originated the transaction, this can differ from the `maker`. for example, if a user is placing offers through a smart contract, the `maker` is the smart contract and the `from_address` is the wallet they are using to interact with the smart contract